specification/vmware/resource-manager/Microsoft.AVS/preview/2019-08-09-preview/examples/PrivateClouds_List.json

```diff
@@ -0,0 +1,58 @@
+{
+  "parameters": {
+    "api-version": "2019-08-09-preview",
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "group1"
+  },
+  "responses": {
+    "200": {
+      "body": [
```

Comment:
>body [](start = 7, length = 4)

Your list response definition is correct but this example doesn't match it. The response in the example should be an object with a `value` array property. You have the same issue in clusters list example. Please fix it there too.